nf_core/modules/modules_command.py

```diff
@@ -62,7 +62,6 @@ def has_valid_directory(self):
         nf_config = os.path.join(self.dir, "nextflow.config")
         if not os.path.exists(main_nf) and not os.path.exists(nf_config):
             raise UserWarning(f"Could not find a 'main.nf' or 'nextflow.config' file in '{self.dir}'")
-        self.has_modules_file()
         return True
```

Comment:
I'm not sure I'm following why you removed this line, and added it as standalone calls.

Reply:
we discussed this on slack:  all commands that used this function use `check_up_to_date` anyway, except for linting. So I kept it there.